framework/src/eventloop/eventloop_common.c

```diff
@@ -0,0 +1,143 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <debug.h>
+#include <unistd.h>
+#include <queue.h>
+#include <eventloop/eventloop.h>
+
+#include "eventloop_internal.h"
+
+el_app_handle_t g_el_app_list[CONFIG_MAX_TASKS];
```

Comment:
When you use any config value as a constant, please use below style to prevent error if config is not set
#ifndef CONFIG_MAX_TASKS
#define CONFIG_MAX_TASKS 64
#endif